activities/trainings/writing-issues.md

```diff
@@ -0,0 +1,87 @@
+---
+type: Guide
+explains: How to clearly communicate what is wrong and suggest fixes
+---
+
+# Writing issues
+
+An issue is a story explaining why something should be changed. It serves as the seed for collaboration in order to get it solved.
+
+Issues can be of two types:
+
+* Bug: something that isn't as it is supposed to be, this is broken
+* Enhancement: an improvement that could be done to make it better
+
+The goal of an issue is to provide knowledge about and context on what to fix or improve. You can make issues alone or together.
+
+An issue can contain a proposed fix, but does not have to.
+
+We write good issues because:
+
+* it helps communicate things that need to be done
+* planning becomes easier if you understand the context of a change
+* the person filing an issue is not always the person to fix it - they might have different skillsets
+* commit messages become better with more context
+
+## Structure
+
+The title of the issue should be short, to the point and descriptive. Enhancements can have verbs in them. Detail, technical or otherwise, should be in the body of the issue.
+
+* Context: why are you filing this issue
+* Problem or idea and this issue's [definition of done](https://about.publiccode.net/organization/definition-of-done.html)
+* Solution or next step: request feedback from others, propose changes or propose further investigation
+
+## Defining done for issues
+
+It can help those who solve the issue to know when they've actually fixed it. If there are multiple factors in this it might help adding them as checkboxes, then they can be resolved and tracked one by one.
```

Comment:
```suggestion
It's helpful for the person who solves the issue to know when they've actually fixed it. List multiple steps or actions with checkboxes so they can be tracked and resolved.
```

Reply:
```suggestion
This helps other contributors know and test whether they have actually fixed it. Issues which require multiple steps, for example, should include actions with checkboxes so these can be tracked and resolved.
```

Reply:
@bvhme - last thing to fix before this can be merged